Tests/Data/vertexing_event_mu20_vertices_AMVF.csv

```diff
@@ -0,0 +1,25 @@
+vertices, posX, posY, posZ, 9 cov entries, nTracks, trkWeight(1), trkComp(1), trkChi2(1)
```

Comment:
Could you name the columns for the covariance entries. To me it is not clear which part (upper/lower) of the covariance is stored and in which order (colum/row-major).

Reply:
Done.